CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.4 (2018-11-30)
+
+- Fixed `SkyListBuilderViewGridsModule` exports to include required submodules and `SkyGridModule`. [#9](https://github.com/blackbaud/skyux-list-builder-view-grids/pull/9)
```

Comment:
Wording/grammar looks fine, but just FYI that I don't see `SkyListBuilderViewGridsModule` anywhere in the related PR, and I don't see it when I open the entire file where we export `SkyGridModule`. So I just want to make sure that the module name is correct.

Reply:
https://github.com/blackbaud/skyux-list-builder-view-grids/pull/9/files#diff-936142aada680d552b8aa57cc92b0e3e

^ this is the module file and it has SkyGridModule in its exports

Reply:
Oh it doesn't actually have builder in the name, nice catch